ember_debug/libs/render-tree.js

```diff
@@ -508,6 +508,7 @@ export default class RenderTree {
         ),
         bounds: this._serializeBounds(node.bounds),
         children: this._serializeRenderNodes(node.children, node),
+        remotes: node.remotes ? this._serializeRenderNodes(node.remotes, node) : undefined,
```

Comment:
Probably just setting this to undefined is enough

Reply:
Or use Object.defineProperty to hide it with enumerable: false. Then in should not be included here

Reply:
https://github.com/emberjs/ember-inspector/blob/5f14672530940fa5b48682783116530f67d58bc7/ember_debug/libs/render-tree.js#L168

Reply:
at line 168
replace
```js
this.currentNode.remotes = this.currentNode.remotes || []; 
```
with
```js
if (!this.currentNode.remotes) Object.defineProperty(this.currentNode, 'remotes', {
  value: []
});
```
Object.defineProperty will make in non-enumerable by default, that way it will not be copied when doing `{...node}`